fr.inria.corese.CoreseGui.yml

```diff
@@ -0,0 +1,70 @@
+app-id: fr.inria.corese.CoreseGui
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+# sdk-extensions: 
+  # - org.freedesktop.Sdk.Extension.openjdk11
+command: /app/bin/run.sh
+modules:
+  - name : openjdk
+    buildsystem: simple
+    build-commands:
+    - /usr/lib/sdk/openjdk11/install.sh
+  - name: corese-gui
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 corese-gui.jar /app/bin/corese-gui.jar
+      - install -Dm755 run.sh /app/bin/run.sh
+      - install -Dm644 fr.inria.corese.CoreseGui.appdata.xml /app/share/metainfo/fr.inria.corese.CoreseGui.appdata.xml
+      - install -Dm644 fr.inria.corese.CoreseGui.desktop /app/share/applications/fr.inria.corese.CoreseGui.desktop
+      - install -Dm644 fr.inria.corese.CoreseGui-512.png /app/share/icons/hicolor/512x512/apps/fr.inria.corese.CoreseGui.png
+      - install -Dm644 fr.inria.corese.CoreseGui-256.png /app/share/icons/hicolor/256x256/apps/fr.inria.corese.CoreseGui.png
+      - install -Dm644 fr.inria.corese.CoreseGui-128.png /app/share/icons/hicolor/128x128/apps/fr.inria.corese.CoreseGui.png
+      - install -Dm644 fr.inria.corese.CoreseGui-64.png /app/share/icons/hicolor/64x64/apps/fr.inria.corese.CoreseGui.png
+      - install -Dm644 fr.inria.corese.CoreseGui-48.png /app/share/icons/hicolor/48x48/apps/fr.inria.corese.CoreseGui.png
+      - install -Dm644 fr.inria.corese.CoreseGui-32.png /app/share/icons/hicolor/32x32/apps/fr.inria.corese.CoreseGui.png
+      - install -Dm644 fr.inria.corese.CoreseGui-24.png /app/share/icons/hicolor/24x24/apps/fr.inria.corese.CoreseGui.png
+      - install -Dm644 fr.inria.corese.CoreseGui-22.png /app/share/icons/hicolor/22x22/apps/fr.inria.corese.CoreseGui.png
+      - install -Dm644 fr.inria.corese.CoreseGui-16.png /app/share/icons/hicolor/16x16/apps/fr.inria.corese.CoreseGui.png
+      - install -Dm644 fr.inria.corese.CoreseGui.svg /app/share/icons/hicolor/scalable/apps/fr.inria.corese.CoreseGui.svg
+    sources:
+      - type: file
+        url: https://github.com/Wimmics/corese/releases/download/release-4.4.1/corese-gui-4.4.1.jar
+        sha256: cb3270d2ceccc9e8194c7d657eeca3c037e957de994f7eca3bbd1cf58fdbea89
+        x-checker-data:
+          - type: json
+          - url: https://api.github.com/repos/wimmics/corese/releases/latest
+          - version-query: .tag_name | gsub("release-";"")
+          - url-query: '"https://github.com/Wimmics/corese/releases/download/release-" + $version + "/corese-gui-" + $version + ".jar"'
+        dest-filename: corese-gui.jar
+      - type: file
+        path: run.sh
+      - type: file
+        path: fr.inria.corese.CoreseGui.appdata.xml
+      - type: file
+        path: fr.inria.corese.CoreseGui.desktop
+      - type: file
```

Comment:
since the icons don't come from upstream, you can remove all of them but  the SVG.